src/gui/filedetails/FileDetailsPage.qml

```diff
@@ -81,7 +81,19 @@ Page {
             Layout.topMargin: root.topPadding
 
             columns: root.showCloseButton ? 3 : 2
-            rows: showFileLockedString ? 3 : 2
+            rows: {
```

Comment:
@claucambra Can you make it shorter? `rows: showFileLockedString && root.fileDetails.fileTagModel.totalTags > 0  ? 3 : 2`?